src/com/zenuml/sequence/plugins/jetbrains/license/CheckLicense.java

```diff
@@ -11,84 +10,80 @@
 import java.security.cert.*;
 import java.util.*;
 
-
-/**
- * @author Eugene Zhuravlev
- * Date: 26-Jul-18
- */
 public class CheckLicense {
   /**
-   * PRODUCT_CODE must be the same specified in plugin.xml inside the <productCode> tag
+   * PRODUCT_CODE must be the same specified in plugin.xml inside the <product-descriptor> tag
    */
-  private static final String PRODUCT_CODE = "PZENUML";
+  private static final String PRODUCT_CODE = "SMPL";
```

Comment:
Why are we changing the PRODUCT_CODE here?

Reply:
changed the code back.